src/cape_ros_node.cpp

```diff
@@ -65,11 +65,11 @@ std::array<uint8_t, 3> hsvToRgb(const double h, const double s, const double v)
 
 CapeRosNode::CapeRosNode(ros::NodeHandle nh)
   : nh_(nh)
-  , cos_angle_max_(std::cos(M_PI / 30))
+  , cos_angle_max_(std::cos(M_PI / 45))
   , max_merge_dist_(50.0)
   , depth_cutoff_(5.0)
   , cylinder_detection_(false)
```

Comment:
`cylinder_detection_` is always false. No extraction of cylinder is done in this code.

Reply:
Yes, we are not using cylinder detection.